torch/_inductor/utils.py

```diff
@@ -1888,7 +1892,7 @@ def device_need_guard(device: str):
 
 def needs_fallback_due_to_atomic_add_limitations(dtype):
     # tl.atomic_add does NOT support the following types
-    return dtype in {torch.int64, torch.bool, torch.bfloat16}
```

Comment:
Wonder if it would be wroth it to allow this pattern - set that is constructed and only ever tested for membership ?